rgw/admin/quota_test.go

```diff
@@ -0,0 +1,38 @@
+package admin
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func (suite *RadosGWTestSuite) TestQuota() {
+	suite.SetupConnection()
+	co, err := New(suite.endpoint, suite.accessKey, suite.secretKey, nil)
+	co.Debug = true
+	assert.NoError(suite.T(), err)
+
+	suite.T().Run("set quota to bucket but quota type is empty", func(t *testing.T) {
+		err := co.SetQuota(context.Background(), Quota{UID: "test"})
+		assert.Error(suite.T(), err)
+		assert.EqualError(suite.T(), err, errMissingQuotaType.Error())
+	})
+
```

Comment:
valid test cases for bucket scoped quota?
Please note while setting `bucket quota` `uid` for the request need to `bucket owner` AFAIR. Otherwise, it won't work in the `radosgw-admin` command. I hope the same applies for `AdminOps` api